tutorial/5. hypercube.ipynb

```diff
@@ -0,0 +1,931 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "min_y = []\n",
+    "max_y = []\n",
+    "for i in range(0, sim_length + 1, time_step_length):\n",
+    "        min_y.append({key: np.min(value) for key, value in concentrations_solved[int(i/time_step_length)].items()})\n",
+    "        max_y.append({key: np.max(value) for key, value in concentrations_solved[int(i/time_step_length)].items()})\n",
+    "time_x = list(map(float, range(0, sim_length + 1, time_step_length)))\n",
+    "\n",
+    "plt.fill_between(time_x, [y['A'] for y in min_y], [y['A'] for y in max_y], alpha = 0.4, label='CONC.A.mol m-3')\n",
+    "plt.fill_between(time_x, [y['B'] for y in min_y], [y['B'] for y in max_y], alpha = 0.4, label='CONC.B.mol m-3')\n",
+    "plt.fill_between(time_x, [y['C'] for y in min_y], [y['C'] for y in max_y], alpha = 0.4, label='CONC.C.mol m-3')\n",
+    "plt.title('Concentration range over time')\n",
+    "plt.ylabel('Concentration (mol m-3)')\n",
+    "plt.xlabel('Time (s)')\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cc0f3ff3",
+   "metadata": {},
+   "source": [
+    "## 7a. Defining User Defined Reactions\n",
```

Comment:
Similarly, I would expect to see a header `7` here, which describes some general information about this section.

Reply:
Also the focus of this section seems to be the user defined reaction. Should it be a separate tutorial rather than combing with LHS tutorial?